src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,26 @@
 const accessibilityQuiz = [
+  {
+    Question: "What is the purpose of the 'aria-labelledby' attribute in ARIA?",
+    Answer:
+      "To enable authors to reference other elements on the page to define an accessible name.",
+    Distractor1: "To hide an element from screen readers",
+    Distractor2: "To add a tooltip to an element",
+    Distractor3: "To change the focus order of elements",
+    Explanation:
+      "The aria-labelledby attribute identifies the element (or elements) that labels the element it is applied to.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby"
```

Comment:
Please consider using the freeCodeCamp resources if it's available.
```suggestion
    Link: "https://forum.freecodecamp.org/t/why-is-aria-labelledby-needed/589633/17"
```